accounts/models.py

```diff
@@ -78,7 +78,7 @@ class Meta:
             ("can_edit_organization", _("Can Edit Organization")),
             ("can_create_organization", _("Can Create Organization")),
             ("can_remove_organization", _("Can Remove Organization")),
-            ("can_view_experimenter", _("Can View Experimenter")),
+            ("can_view_researcher", _("Can View Researcher")),
```

Comment:
this we can leave since it's referencing the application ("can view the experimenter app" rather than "can view a particular researcher") - & then won't need migration above